pyproject.toml

```diff
@@ -23,20 +23,20 @@ cdiserrors = "<2.0.0"
 cdislogging = "^1.0.0"
 cdispyutils = "^2.0.1"
 cryptography = ">=3.2"
-flask = ">=1.1.1"
+flask = ">=2.2.3"
 flask-cors = "^3.0.3"
 flask-restful = "^0.3.6"
 flask_sqlalchemy_session = "^1.1"
```

Comment:
you can remove `flask_sqlalchemy_session`. It should have been removed from everywhere. Just do a search to confirm no other file is using it.